crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/generated_acir.rs

```diff
@@ -368,7 +368,7 @@ impl GeneratedAcir {
     /// n.b: we do NOT check here that lhs and rhs are indeed 'bits' size
     /// lhs < rhs <=> a+1<=b
     /// TODO: Consolidate this with bounds_check function.
-    fn bound_constraint_with_offset(
+    pub(crate) fn bound_constraint_with_offset(
```

Comment:
Would prefer to keep this private and we wrap it in the higher level functionality we need. 

For example, in this PR, we are using it to do a less_than, in that case, we should expose a less_than function. This is if we exclude the fact that we already have a less_than_var method

Reply:
Done